conf/messages

```diff
@@ -212,12 +212,16 @@ chargeType.OADefaultInterestTitle = VAT officer’s assessment interest
 chargeType.OADefaultInterestDescription = interest charged on the officer’s assessment
 chargeType.bnpRegPre2010ChargeTitle = Penalty for late registration
 chargeType.bnpRegPre2010ChargeDescription = because you should have been registered for VAT earlier
+chargeType.bnpRegPre2010ChargeDescription.agent = because your client should have been registered for VAT earlier
 chargeType.bnpRegPost2010ChargeTitle = Penalty for late registration
 chargeType.bnpRegPost2010ChargeDescription = because you should have been registered for VAT earlier
+chargeType.bnpRegPost2010ChargeDescription.agent = because your client should have been registered for VAT earlier
 chargeType.ftnMatPre2010ChargeTitle = Failure to notify penalty
-chargeType.ftnMatPre2010ChargeDescription = you did not tell us you are no longer exempt from VAT registration
+chargeType.ftnMatPre2010ChargeDescription = because you did not tell us you are no longer exempt from VAT registration
+chargeType.ftnMatPre2010ChargeDescription.agent = because your client did not tell us they we were no longer exempt from VAT registration
```

Comment:
This line has the same typo as line 224. Could you please fix this as well? Apologies I missed this before